fulp_modules/features/antagonists/bloodsuckers/code/powers/feed.dm

```diff
@@ -131,7 +133,28 @@
 	var/mob/living/user = owner
 	var/mob/living/feed_target = target_ref.resolve()
 	if(!ContinueActive(user, feed_target))
-		owner.balloon_alert(owner, "interrupted!")
+		if(silent_feed)
```

Comment:
When is this decided? It's not always silent

Reply:
it got lost in translation